pkg/virt-controller/watch/vmi.go

```diff
@@ -2135,3 +2154,78 @@ func (c *VMIController) handleDynamicInterfaceRequests(vmi *virtv1.VirtualMachin
 
 	return nil
 }
+
+func (c *VMIController) updateInterfaceStatus(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod) error {
+	originalVMIStatus := vmi.Status.DeepCopy().Interfaces
+	indexedVMIStatus := indexIfaceStatus(originalVMIStatus)
+
+	var newStatus []virtv1.VirtualMachineInstanceNetworkInterface
+	ifaceNamingScheme := namescheme.CreateNetworkNameScheme(vmi.Spec.Networks)
+	for _, network := range vmi.Spec.Networks {
+		status := virtv1.VirtualMachineInstanceNetworkInterface{Name: network.Name}
+		if _, ok := indexedVMIStatus[network.Name]; ok {
+			// Already have the status, modify if needed
+			status = indexedVMIStatus[network.Name]
+		}
+
+		podIfaceName, wasFound := ifaceNamingScheme[network.Name]
+		if !wasFound {
+			return fmt.Errorf("could not find the pod interface name for network [%s]", network.Name)
+		}
+
+		indexedPodIfaces := nonDefaultMultusNetworksIndexedByIfaceName(pod)
+		if _, found := indexedPodIfaces[podIfaceName]; found {
+			status.PodConfigDone = true
```

Comment:
Please update the commit message which using the term `PodIfaceReady`. Also it says something about`PodInraReady`, couldn't find it in the commit.

Reply:
Yes, you're right. That was the initial name I am now using for `PodIfaceReady` .

Thanks for ensuring the cohesiveness of the PR, including the commit msgs.

Reply:
The commit message is still outdated.

Reply:
Done.